docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -918,6 +918,13 @@ To collect statistics for a table, execute the following statement::
 
   ANALYZE table_schema.table_name;
 
+To recalculate from scratch the statistics for the table use additional parameter ``mode``.
```

Comment:
Please replace the `.` with `:`. Otherwise, the highlight doesn't work in the following SQL.